examples/nodejs_etcd_operator/README.md

```diff
@@ -0,0 +1,46 @@
+## Binding an Imported app to an In-cluster Operator Managed Etcd Database.
+
+1. Install Etcd operator using operator hub, 
+   follow https://operatorhub.io/operator/etcd
+2. Create a Etcd cluster.
+ ```
+ apiVersion: "etcd.database.coreos.com/v1beta2"
+ kind: "EtcdCluster"
+ metadata:
+  name: "etcd-cluster"
+ spec:
+  size: 5
+  version: "3.2.13"
+ ```
+3. Deploy application which uses Etcd client lib using openshift 4 devconsole.
+Test application : https://github.com/akashshinde/node-todo.git
+![](https://i.imgur.com/WGQZ1nj.png)
+
+4. Create SBR.
+```
```

Comment:
Could you add [syntax highlight](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for `yaml`?